### DISCUSSION TITLE
unable to dismiss reminders

### DISCUSSION THREAD

**Discussion by avners** [kewisch/gdata-provider#530] [Support] [answered]
Hello

I have been using Provider for google calendar for about 20 years now, Thanks!

Lately (not sure exactly when it started, about a couple of weeks ) I am unable to dismiss reminders
whenever I try to dismiss a reminder the google access check window is opened and immediately closed without asking me for the password
I am using version 102.4.2 (32-bit)
 
 Thanks!
 Avner

**Comment by xjonx1** [comment]
The app has to be updated. The developer has been contacted. The problem is the app needs googles new OAuth 2.0.

**Comment by avners** [comment]
thanks!

**Comment by xjonx1** [comment]
According to his twitter, he is currently involved in some Linux conference right now.

**Comment by avners** [comment]
thanks
will wait patiently

**Comment by avners** [comment]
thank you for fixing this much needed feature

**Comment by xjonx1** [comment, marked as answer]
There was an issue with the add-on and is now fixed by the developer. Details here:https://github.com/kewisch/gdata-provider/discussions/519#discussioncomment-4147395
